Source/BoundaryConditions/WarpX_PEC.cpp

```diff
@@ -333,33 +333,44 @@ namespace
+     *             added to its mirror location inside the domain.
+     *             This can be interpreted as depositing the charge/current from the oppositely
+     *             charged mirror particle inside the PEC. It can also be interpreted as using
+     *             a lower order shape factor for the particles near the boundary.
+     *        PMC: The charge and current density parallel to the boundary deposited to the
+     *             guard cells is added to its mirror location inside the domain.
+     *             The current density normal to the boundary deposited to the guard cells is
+     *             subtracted from its mirror location inside the domain. Since PMC is a
+     *             symmetry boundary, this charge and current comes from the physical
+     *             mirror particle with the same charge on the other side of the boundary.
      *
-     * \param[in] n                 index of the MultiFab component being updated
-     * \param[in] ijk_vec           indices along the x(i), y(j), z(k) of the rho Array4
-     * \param[in out] field         field data to be updated
-     * \param[in] mirrorfac         mirror cell is given by mirrorfac - ijk_vec
-     * \param[in] psign             Whether the field value should be flipped across the boundary
-     * \param[in] is_reflective     Whether the given particle boundary is reflecting or field boundary is pec
-     * \param[in] tangent_to_bndy   Whether a given direction is perpendicular to the boundary
-     * \param[in] fabbox            multifab box including ghost cells
+     * \param[in] n                  index of the MultiFab component being updated
+     * \param[in] ijk_vec            indices along the x(i), y(j), z(k) of the rho Array4
+     * \param[in out] field          field data to be updated
+     * \param[in] mirrorfac          mirror cell is given by mirrorfac - ijk_vec
+     * \param[in] psign              Whether the field value should be flipped across the boundary
+     * \param[in] is_pec_pmc_bndy    Whether the field boundary is pec or pmc
+     * \param[in] tangent_to_bndy    Whether a given direction is perpendicular to the boundary
+     * \param[in] fabbox             multifab box including ghost cells
      */
     AMREX_GPU_DEVICE AMREX_FORCE_INLINE
     void SetRhoOrJfieldFromPEC (const int n,
                                 const amrex::IntVect & ijk_vec,
                                 amrex::Array4<amrex::Real> const& field,
                                 amrex::GpuArray<GpuArray<int, 2>, AMREX_SPACEDIM> const& mirrorfac,
                                 amrex::GpuArray<GpuArray<amrex::Real, 2>, AMREX_SPACEDIM> const& psign,
-                                amrex::GpuArray<GpuArray<int, 2>, AMREX_SPACEDIM> const& is_reflective,
+                                amrex::GpuArray<GpuArray<int, 2>, AMREX_SPACEDIM> const& is_pec_pmc_bndy,
```

Comment:
I guess since this is updated, we might as well switch it to `bool`.
```suggestion
                                amrex::GpuArray<GpuArray<bool, 2>, AMREX_SPACEDIM> const& is_pec_pmc_bndy,
```